Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -2223,6 +2223,18 @@ def populate_url_filter_category_from_context(category):
             return context_urls
 
 
+def calculate_dbot_score(url, category):
+    dbot_score = 1
```

Comment:
why is `1` the default and not `0` ?

Reply:
Not really, i'll need security help.
I'm sure the categoris list I have here is partial.